src/Symfony/Component/Validator/Context/ExecutionContext.php

```diff
@@ -310,6 +318,8 @@ public function getPropertyPath($subPath = '')
      */
     public function addViolationAt($subPath, $message, array $parameters = array(), $invalidValue = null, $plural = null, $code = null)
     {
+        trigger_error('The '.__METHOD__.' is deprecated since version 2.5 and will be removed in 3.0. Use the '.__CLASS__.'::buildViolation method instead.', E_USER_DEPRECATED);
```

Comment:
should be removed here, given we already throw a BadMethodCallException saying this method does not exist in this version of the API (it is not a deprecation in this case, because the method is not working at all. The deprecation is in the child class only)